<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QWidget" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string/>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="FormDati">
      <property name="toolTip">
       <string/>
      </property>
      <attribute name="title">
       <string>Form inserimento dati</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="gStella">
           <property name="title">
            <string>Stella</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="2" column="0">
                 <widget class="QLabel" name="sForza">
                  <property name="text">
                   <string>Forza di gravità</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDoubleSpinBox" name="sTemperaturaBox">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="sNomeBox">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="sDiametroBox">
                  <property name="wrapping">
                   <bool>false</bool>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="showGroupSeparator" stdset="0">
                   <bool>false</bool>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="sNome">
                  <property name="text">
                   <string>Nome</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDoubleSpinBox" name="sForzaBox">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="sTemperatura">
                  <property name="text">
                   <string>Temperatura</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="sDiametro">
                  <property name="text">
                   <string>Diametro</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="sSubmit">
                <property name="text">
                 <string>Submit</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gPianeta">
           <property name="title">
            <string>Pianeta</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="4" column="1">
                 <widget class="QComboBox" name="pTipoBox">
                  <item>
                   <property name="text">
                    <string>roccioso</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>gassoso</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="pDiametro">
                  <property name="text">
                   <string>Diametro</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="pForza">
                  <property name="text">
                   <string>Forza di gravità</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDoubleSpinBox" name="pDiametroBox">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDoubleSpinBox" name="pForzaBox">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="pTipo">
                  <property name="text">
                   <string>Tipo</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="pNome">
                  <property name="text">
                   <string>Nome</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="pNomeBox"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="pSistema">
                  <property name="text">
                   <string>Sistema</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="pSistemaBox"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="pSubmit">
                <property name="text">
                 <string>Submit</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gSatellite">
           <property name="title">
            <string>Satellite</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="1" column="1">
                 <widget class="QLineEdit" name="saNomeBox"/>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="saOrbitatoBox"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="saNome">
                  <property name="text">
                   <string>Nome</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDoubleSpinBox" name="saForzaBox">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="saOrbitato">
                  <property name="text">
                   <string>Pianeta orbitato</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="saDiametro">
                  <property name="text">
                   <string>Diametro</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDoubleSpinBox" name="saDiametroBox">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="saForza">
                  <property name="text">
                   <string>Forza di gravità</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QComboBox" name="saTipoBox">
                  <item>
                   <property name="text">
                    <string>roccioso</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>gassoso</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="saTipo">
                  <property name="text">
                   <string>Tipo</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="saSistema">
                  <property name="text">
                   <string>Sistema</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="saSistemaBox"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="saSubmit">
                <property name="text">
                 <string>Submit</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Database">
      <attribute name="title">
       <string>Database</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="1" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string>Sistema stellare</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Dato</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
